EventKit/eventkit-cloud
# adds projection source for all export tasks

# Description of Improvement

Adds src_prj as explicit default to export tasks, fixes issue where None is sometimes passed. 

## How to test

Create exports with different formats and projections.  Ensure that they pass successfully. 

## Quality Assurance

The following items have been updated:


## Screenshots



:smile:
